test/functional/services/test_subjects.ts

```diff
@@ -55,9 +55,12 @@ export function TestSubjectsProvider({ getService }: FtrProviderContext) {
       selector: string,
       existsOptions?: ExistsOptions
     ): Promise<void | never> {
-      if (!(await this.exists(selector, { timeout: TRY_TIME, ...existsOptions }))) {
-        throw new Error(`expected testSubject(${selector}) to exist`);
-      }
+      const options = { timeout: TRY_TIME, ...existsOptions };
+      await retry.tryForTime(options.timeout, async () => {
+        if (!(await this.exists(selector, options))) {
```

Comment:
@pheyos I'm not sure that this way we actually fix the problem.

```
may fail after a few milliseconds, if myTestSubject is found that fast but is not visible yet.
In order to make the whole existOrFail operation follow the provided timeout, this PR adds a retry.
```

I think the proper way is to fix the root cause: `find.existsByDisplayedByCssSelector`.

Maybe we should use default timeout on Webdriver side to make find + isDispayed operations more frequent. 
And timeout should be used for retry service as an indicator of exit. Thoughts?

Reply:
Makes sense to fix it in `find.existsByDisplayedByCssSelector`, I will push changes

Reply:
Done in b284bb9